packages/app/src/components/css-reset.tsx

```diff
@@ -0,0 +1,344 @@
+import React from 'react';
+import { Global, css } from '@emotion/react';
+import { theme } from '@stacks/ui-theme';
+
+const defaultConfig = (theme: any) => ({
+  light: {
+    color: theme.colors.ink[900],
+    bg: undefined as any,
+    borderColor: '#E5E5EC', // TODO: replace this with theme color
+    placeholderColor: theme.colors.ink[400],
+  },
+  dark: {
+    color: 'white',
+    bg: theme.colors.ink[900],
+    borderColor: theme.colors.ink[600],
+    placeholderColor: theme.colors.ink[500],
+  },
+});
+const { color, bg, borderColor, placeholderColor } = defaultConfig(theme).light;
+
+const cssReset = css`
```

Comment:
Can't you just impor some normalize css package, rather than coping it all? 🤔

Reply:
yeah we should be able to use the one from `@stacks/ui`